src/main/java/net/sf/mzmine/modules/visualization/ida/IDADataSet.java

```diff
@@ -273,7 +273,7 @@ public void setColor(int series, int item, Color c) {
      * 
      */
     public void highlightSpectra(double mz, double ppm, int minIntensity,
```

Comment:
This comment is not related to this commit, but I wonder why you used double mz and double ppm in the constructor. Other modules use the MZTolerance class specifically for this purpose.